ouroboros-consensus/src/Ouroboros/Consensus/Mempool/Impl.hs

```diff
@@ -185,21 +193,29 @@ implSnapshotGetTx IS{isTxs} tn = isTxs `lookupByTicketNo` tn
 
 data ValidationResult blk = ValidationResult {
     -- | The tip of the chain before applying these transactions
-    vrBefore  :: ChainHash blk
+    vrBefore       :: ChainHash blk
 
     -- | The transactions that were found to be valid (oldest to newest)
-  , vrValid   :: TxSeq (GenTxId blk, GenTx blk)
+  , vrValid        :: TxSeq (GenTxId blk, GenTx blk)
```

Comment:
Can we remove `GenTxId blk` here too?

I think we should also add something like the following comment to the API:

```diff
   -- | Given a 'GenTx', compute its 'GenTxId'.
++ --
++ -- Should be cheap, as this will be called often.
   computeGenTxId :: GenTx blk -> GenTxId blk
```